end-to-end-tests/docker-compose.yaml

```diff
@@ -0,0 +1,85 @@
+version: '2.4'
```

Comment:
Please move this file to the `$PROJECT_DIR/docker-compose` directory.

Reply:
Done